WordPress/gutenberg
# [Mobile] - KeyboardAvoidingView - Set a background color for the toolbar container

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/4370

- `Gutenberg Mobile PR` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/4749

## What?
It sets a background color for the toolbar container within the `KeyboardAvoidingView` **on iOS**.

## Why?
On iPad devices when there's an external keyboard being used, a gap between the toolbar and the bottom of the screen is visible.

## How?
By setting a background color to the toolbar container, this way when there's an external keyboard is being used, it won't look like the UI is broken.

## Testing Instructions

- Open the app using an iPad simulator or device
- Add a Paragraph block
- Type some text
- **Expect to not see** a gap between the toolbar and the bottom of the screen.

## Screenshots or screencast 

Dark mode (before) | Dark mode (after)
-|-
<kbd><img src="https://user-images.githubusercontent.com/4885740/162941975-742a76d7-5b53-439e-98dc-7446158d3f24.png" width="400" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/4885740/162942324-aeda1fe3-ef36-4ff5-90f2-57e99a04cc71.png" width="400" /></kbd>

Light mode (before) | Light mode (after)
-|-
<kbd><img src="https://user-images.githubusercontent.com/4885740/162942685-fc2b5cb0-d606-4615-a5c8-0fe38cb5e00e.png" width="400" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/4885740/162942636-a8d642c3-b13f-40ab-a036-81f289d443ea.png" width="400" /></kbd>
